src/lib/bootloader/grub2_widgets.rb

```diff
@@ -308,7 +308,7 @@ def contents
             HBox(
               HSpacing(2),
               # text entry
-              Password(Id(:pw1), Opt(:hstretch), _("&Password")),
+              Password(Id(:pw1), Opt(:hstretch), _("&Password for GRUB2 User 'root'")),
```

Comment:
OK, will add it. I think something like keep it short is enough as problem is only in ncurses in small window